### PR TITLE
2.0 preview

### PR DESCRIPTION
**Pull request by e111077**
DO NOT MERGE

### CODE REVIEW

File: iron-doc-property.html [hunk 1]
```diff
@@ -182,8 +182,8 @@
         },
 
         // hidden if no params, and no return value
-        _computeHideParams: function(descriptor,ret) {
-          return (!descriptor.params || descriptor.params.length === 0) && !ret;
```
Comment (notwaldorf) [inline iron-doc-property.html]: why no more `ret`?

File: test/index.html [hunk 2]
```diff
@@ -18,8 +18,8 @@
   <body>
     <script>
       WCT.loadSuites([
-        'iron-doc-viewer.html',
-        'iron-doc-viewer.html?dom=shadow'
+        'iron-doc-viewer.html?dom=shadow',
+        'iron-doc-viewer.html?wc-shadydom=true&wc-register=true'
```
Comment (notwaldorf) [inline test/index.html]: `wc-ce` 🎏

File: test/iron-doc-viewer.html [hunk 3]
```diff
@@ -34,9 +34,22 @@
...
+            is: 'bound-fixture'
+          });
+        });
+      </script>
+    </dom-module>
+
     <test-fixture id="bound">
-      <template is="dom-template">
-        <iron-doc-viewer descriptor="{{descriptor}}" bound></iron-doc-viewer>
+      <template>
```
Comment (notwaldorf) [inline test/iron-doc-viewer.html:51]: why did this need to change? ohhh no `dom-bind`. right? cool.

**Review by notwaldorf — COMMENTED** [review]
just some rando questions

File: iron-doc-property.html (hunk 1 above)
Comment (e111077) [reply iron-doc-property.html]: `this.return` was never actually ever defined and this function was actually never being called in 1.0, fo real. After looking back at this, I believe that they meant to bind `descriptor.return`. will update

File: test/iron-doc-viewer.html (hunk 3 above)
Comment (e111077) [reply test/iron-doc-viewer.html:51]: this was implemented before dom-template was brought in. Do you still want me to change it back?

Comment (notwaldorf) [reply test/iron-doc-viewer.html:51]: nah i think it's fine

**Review by notwaldorf — APPROVED** [review]

**Pull request merged**